chain/cosmos/cosmos_chain.go

```diff
@@ -134,52 +135,43 @@ func (c *CosmosChain) GetAddress(ctx context.Context, keyName string) ([]byte, e
 
 // Implements Chain interface
 func (c *CosmosChain) SendFunds(ctx context.Context, keyName string, amount ibc.WalletAmount) error {
-	return c.getRelayerNode().SendFunds(ctx, keyName, amount)
+	return c.getFullNode().SendFunds(ctx, keyName, amount)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) SendIBCTransfer(ctx context.Context, channelID, keyName string, amount ibc.WalletAmount, timeout *ibc.IBCTimeout) (string, error) {
-	return c.getRelayerNode().SendIBCTransfer(ctx, channelID, keyName, amount, timeout)
+	return c.getFullNode().SendIBCTransfer(ctx, channelID, keyName, amount, timeout)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) InstantiateContract(ctx context.Context, keyName string, amount ibc.WalletAmount, fileName, initMessage string, needsNoAdminFlag bool) (string, error) {
-	return c.getRelayerNode().InstantiateContract(ctx, keyName, amount, fileName, initMessage, needsNoAdminFlag)
+	return c.getFullNode().InstantiateContract(ctx, keyName, amount, fileName, initMessage, needsNoAdminFlag)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) ExecuteContract(ctx context.Context, keyName string, contractAddress string, message string) error {
-	return c.getRelayerNode().ExecuteContract(ctx, keyName, contractAddress, message)
+	return c.getFullNode().ExecuteContract(ctx, keyName, contractAddress, message)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) DumpContractState(ctx context.Context, contractAddress string, height int64) (*ibc.DumpContractStateResponse, error) {
-	return c.getRelayerNode().DumpContractState(ctx, contractAddress, height)
+	return c.getFullNode().DumpContractState(ctx, contractAddress, height)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) ExportState(ctx context.Context, height int64) (string, error) {
-	return c.getRelayerNode().ExportState(ctx, height)
+	return c.getFullNode().ExportState(ctx, height)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) CreatePool(ctx context.Context, keyName string, contractAddress string, swapFee float64, exitFee float64, assets []ibc.WalletAmount) error {
-	return c.getRelayerNode().CreatePool(ctx, keyName, contractAddress, swapFee, exitFee, assets)
-}
-
-// Implements Chain interface
-func (c *CosmosChain) WaitForBlocks(number int64) (int64, error) {
-	return c.getRelayerNode().WaitForBlocks(number)
-}
-
-func (c *CosmosChain) Height() (int64, error) {
-	return c.getRelayerNode().Height()
+	return c.getFullNode().CreatePool(ctx, keyName, contractAddress, swapFee, exitFee, assets)
 }
 
 // Implements Chain interface
 func (c *CosmosChain) GetBalance(ctx context.Context, address string, denom string) (int64, error) {
 	params := &bankTypes.QueryBalanceRequest{Address: address, Denom: denom}
-	grpcAddress := dockerutil.GetHostPort(c.getRelayerNode().Container, grpcPort)
+	grpcAddress := dockerutil.GetHostPort(c.getFullNode().Container, grpcPort)
```

Comment:
The name "relayer node" didn't really make sense to me. It's not a term you find in cosmos or tendermint. So I changed it. 

FWIW, I don't like `full node` either because I think "full" means it does everything, including validate. But "full" is used a lot in this space.

Reply:
I think the relayer node here really meant, the full node that the relayer is configured to connect to. `getFullNode` doesn't really capture that it will always return the same full node, but `getRelayerNode` wasn't intuitive either. I don't have a better name suggestion offhand yet.

Reply:
Tough one to name here for sure. For me `relayerNode` on the surface meant the process in which the relayer itself was running. In this case, it means the chain process from which the relayer can query.